distribution/zip/ballerina/src/assembly/bin.xml

```diff
@@ -87,6 +87,12 @@
             </directory>
             <outputDirectory>src</outputDirectory>
         </fileSet>
+        <fileSet>
```

Comment:
I think this doesn't work anymore, because jar file doesn't contain source bal files anymore